src/test.wtest

```diff
@@ -0,0 +1,55 @@
+import casa.*
+
+test " Aumenta el monto de reparación a $1000: hay víveres suficientes, hay que hacer reparaciones 
+y la casa no está en orden" {
+	casaDePepeYJulian.aumentaGastoEnReparaciones(1000)
+	assert.equals(true, casaDePepeYJulian.viveresSuficientes())

```

Comment:
Esta bien armado este test , y esta bien lo que queres chequear . Pero no es correcto usar un assert.equals() para un mensaje que te devuelve un True o un False . Recorda que para esos casos podes usar el assert.that (objeto.mensaje())